src/app/static/src/app/store/translations/locales.ts

```diff
@@ -188,6 +211,52 @@ const en: Translations = {
     ART: "ART",
     apiCouldNotParseError: "Could not parse API response. Please contact support.",
     apiMissingError: "API response failed but did not contain any error information. Please contact support.",
+    axe: "Accessibility",
+    axeOnNaomiHeading: "Accessibility on Naomi",
+    axeOnNaomiParagraph1: "This statement applies to content published on <a href=\"https://naomi.unaids.org/\"> naomi.unaids.org </a>",
+    axeOnNaomiParagraph2: "We want as many people as possible to be able to use this website. For example, that means you should\n" +
+        "                    be able to:",
+    axeOnNaomiPurposeList: "<li>Resize your window with content being reformatted appropriately</li>\n" +
```

Comment:
Emma mentioned a preference for not including html code here. I've no issue it with myself but just thought I'd check it was ok in this case.

Reply:
i don't mind some light HTML inclusions, I was just objecting to really long, highly structured HTML being expressed as strings. I think this sort of level is ok.